tests/all/unit/api/validation/test_createdat.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from app.models import db
+from tests.all.integration.utils import OpenEventTestCase
+from app.api.helpers.db import save_to_db
+from datetime import datetime,  timedelta, timezone
+from app.models.user import User
+from app.models.event import Event
+from app.models.access_code import AccessCode
+from app.models.order import Order
+from app.models.session import Session
+from app.models.ticket_holder import TicketHolder
+from app.models.role_invite import RoleInvite
+from app.models.discount_code import DiscountCode
+from app.models.event_invoice import EventInvoice
+
+
+class TestCreatedatValidation(OpenEventTestCase):
+    def test_createdat(self):
+        """ Validate time : Tests if created_at is set to current time in all models
+        """
+        with self.app.test_request_context():
+            test_user = User(email='authtest@gmail.com', password='password')
+            test_event = Event(
+                name='test event', starts_at=datetime.now(),
+                ends_at=datetime.now()+timedelta(days=1),
+                logo_url='foo.com')
+            test_session = Session(title='Test Session')           
+            test_access_code = AccessCode(code=123)
+            test_order = Order()
+            test_ticket_holder = TicketHolder(
+                firstname="John",
+                lastname="Snow")
+            test_role_invite = RoleInvite(
+                email="testrole@mail.com",
+                role_name="Nothing")
+            test_discount_code = DiscountCode(
+                code="123", value=0.0,
+                type="test", used_for="nothing")
+            test_event_invoice = EventInvoice()
```

Comment:
Use factories and remove initializing logic

Reply:
can I create factories for models that don't have one yet?

Reply:
Yes